creusot/src/backend/clone_map.rs

```diff
@@ -111,7 +109,7 @@ pub struct CloneMap<'tcx> {
     clone_level: CloneLevel,
 
     // DefId of the item which is cloning. Used for trait resolution
-    self_id: DefId,
+    self_id: CloneNode<'tcx>, // TODO should this be a TransId?
```

Comment:
Is this `TODO` still relevant?

Reply:
Yes, this is a cleanup I still want to do.